OI/Hipster userland patch for libgd-2.0.35 recipe.

Changes needed to get gd2 tests working.

This change needs to be further evaluated to see if it should be passed
upstream.

--- tests/gdimageline/CMakeLists.txt.orig	2014-12-10 16:08:04.845700212 -0800
+++ tests/gdimageline/CMakeLists.txt	2014-12-10 16:07:38.129451205 -0800
@@ -7,6 +7,6 @@
 
 FOREACH(test_name ${TESTS_FILES})
 	add_executable(${test_name} "${test_name}.c")
-	target_link_libraries (${test_name} gdTest ${GD_LIB})
+	target_link_libraries (${test_name} gdTest ${GD_LIB} "-lm")
 	ADD_TEST(${test_name} ${EXECUTABLE_OUTPUT_PATH}/${test_name})
 ENDFOREACH(test_name)
